LayoutTests/fast/mediastream/video-srcObject-set-twice.html

```diff
@@ -22,6 +22,12 @@
    return pixel.r === 0 && pixel.g === 128 && pixel.b === 0;
 }
 
+function isPixelGreen(x, y, canvas, data)
```

Comment:
the name isn't particularly suitable nor represent what it's doing, it's is isPixelAlmostGreen